package.json

```diff
@@ -12,19 +12,20 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+ssh://git@github.com/gemini-testing/hermione-retry-progressive.git"
+    "url": "git+ssh://git@github.com/gemini-testing/testplane-retry-progressive.git"
   },
   "keywords": [
+    "testplane",
     "hermione",
```

Comment:
did you decide to leave it on purpose? Maybe it makes sense

Reply:
> did you decide to leave it on purpose?

Yes, because the code base is the same.